src/runner.rs

```diff
@@ -36,7 +37,7 @@ pub const DEFAULT_CALL_STACK_LIMIT: usize = 64 * 1024;
 /// at these boundaries.
 #[derive(Copy, Clone, Debug, PartialEq, Default)]
 #[repr(transparent)]
-struct RuntimeValueInternal(pub u64);
+pub struct RuntimeValueInternal(pub u64);
```

Comment:
Why does this need to be public?

Reply:
```rust
pub fn with_stacks(
    value_stack: StackWithLimit<RuntimeValueInternal>,
    call_stack: StackWithLimit<FunctionContext>,
)
```

`RuntimeValueInternal` is in the type signature for `with_stacks`. rustc won't let us make `with_stacks` public unless `RuntimeValueInternal` is also public. This is what happens when `RuntimeValueInternal` is made private:

```
error[E0446]: private type `runner::RuntimeValueInternal` in public interface
```

Reply:
Made `RuntimeValueInternal` private in b76ad46.